ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -1179,12 +1177,12 @@ export const DatasetPage = fp.flow(
             if (
               (PREPACKAGED_DOMAINS[prepackaged] === Domain.SURVEY &&
                 domainWithConceptSetId.domain === Domain.SURVEY &&
-                domainWithConceptSetId.conceptSetId === null &&
+                domainWithConceptSetId.conceptSetId === undefined &&
```

Comment:
The fields in the `Select Values (Columns)` section were remaining after deselecting a prepackaged concept set. Seems to be working as expected after changing the `null` checks to `undefined`